i18n/translationProvider.go

```diff
@@ -50,6 +53,14 @@ func (tp *TranslationProvider) Update(d *deps.Deps) error {
 
 	for _, currentSource := range sources {
```

Comment:
Add a comment here that we should plural specs of languages to prevent the error of using custom language codes.
And add a link to #3564 too.